railties/lib/rails/generators/rails/app/templates/config/initializers/hsts_subdomains.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.ssl_options.reverse_merge!({ hsts: { subdomains: true } })
```

Comment:
@rafaelfranca Made the change as per our discussion. This will make sure that we don't mess up with user's `config.ssl_options[:hsts]`. It can be false or true or whatever, it will still remain that value after this initializer will get run.